contracts/FractalModule.sol

```diff
@@ -22,16 +22,16 @@ contract FractalModule is IFractalModule, Module {
         __Ownable_init();
         (
             address _owner, // Controlling DAO
-            address _avatar, // GSafe // Address(0) == msg.sender
-            address _target, // GSafe or Modifier  // Address(0) == msg.sender
+            address _avatar,
+            address _target, 
             address[] memory _controllers // Authorized controllers
         ) = abi.decode(
                 initializeParams,
                 (address, address, address, address[])
             );
 
-        setAvatar(_avatar == address(0) ? msg.sender : _avatar);
-        setTarget(_target == address(0) ? msg.sender : _target);
```

Comment:
why was this necessary previously, or was it?

Reply:
This was necessary to support the callback method in order to pass in the gnosis safe address. However now, it is no longer needed because we can predict the address before setting up the module.